src/persist/src/persister.rs

```diff
@@ -18,7 +18,6 @@ pub trait Snapshot {
     fn read<E: Extend<((String, String), u64, isize)>>(&mut self, buf: &mut E) -> bool;
 
     /// A full read of the data in the snapshot.
-    #[cfg(test)]
```

Comment:
its a bummer to remove the test label from this function -- not sure if I have to do that

Reply:
kept the test label and made a local copy inside the benchmark